submissions/NatalyaBortsova/popup/index.html

```diff
@@ -0,0 +1,175 @@
+                  <label class="menu__item-label menu__item-label_more" for="more">More</label>
+                  <ul class="popup__list popup__list_more">
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/photos.png" alt="Photos" width="50" height="50" />
+                        <span class="popup__name">Photos</span>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/promo.png" alt="Promo" width="50" height="50" />
+                        <span class="popup__name">Promo</span>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/chat.png" alt="Translation" width="50" height="50" />
+                        <span class="popup__name">Translation</span>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/videos.png" alt="Videos" width="50" height="50" />
+                        <span class="popup__name">Videos</span>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/blogs.png" alt="Blogs" width="50" height="50" />
+                        <span class="popup__name">Blogs</span>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/search.png" alt="Search" width="50" height="50" />
+                        <span class="popup__name">Search</span>
+                      </a>
+                    </li>
+                  </ul>
+                  <a href="#" class="popup__bottom-link">
+                    Even more from Kottans
+                  </a>
+                </div>
+              </div>
+            </div>
+          </li>
+          <li class="menu__item">
+            <button class="menu__item-button animation" type="button" tabindex="0">
+              <img class="menu__image" src="images/icons/bell.png" alt="bell" width="25" height="25" />
+            </button>
+          </li>
+          <li class="menu__item">
+            <button class="menu__item-button animation" type="button" tabindex="0">
+              <img class="menu__image" src="images/icons/user.png" alt="user photo" width="30" height="30" />
```

Comment:
The problem with `alt` attribute content is still not fixed. :cry:  [Read this article](https://htmlacademy.ru/blog/articles/alt-text)